### PR TITLE
Add missing French and English translations.

### PR DESCRIPTION
**Pull request by njohner**
For [CA-2646]

## Checklist

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [ ] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

[CA-2646]: https://4teamwork.atlassian.net/browse/CA-2646

### CODE REVIEW

File: opengever/meeting/locales/en/LC_MESSAGES/opengever.meeting.po [hunk 1]
```diff
@@ -1828,12 +1828,12 @@ msgstr "Memberships"
 #. Default: "Attachment order updated."
 #: ./opengever/meeting/browser/meetings/meeting.py
 msgid "message_attachment_order_successfully_message"
-msgstr ""
+msgstr "The attachments'order has been successfully updated"
```
Comment (lukasgraf) [inline opengever/meeting/locales/en/LC_MESSAGES/opengever.meeting.po:1831]: `attachment's order`

**Review by lukasgraf — APPROVED** [review]
Just a small typo, LGTM otherwise 👍

File: opengever/meeting/locales/en/LC_MESSAGES/opengever.meeting.po (hunk 1 above)
Comment (njohner) [reply opengever/meeting/locales/en/LC_MESSAGES/opengever.meeting.po:1831]: No I think that is correct. `attachments` is plural here, so possession is simply an apostrophe.

Comment (lukasgraf) [reply opengever/meeting/locales/en/LC_MESSAGES/opengever.meeting.po:1831]: Oh. I was considering that, but wasn't quite clear. But then there's a space missing, sorry ;)

Comment (njohner) [reply opengever/meeting/locales/en/LC_MESSAGES/opengever.meeting.po:1831]: ah yes 😅

**Pull request merged**